src/cmark.h

```diff
@@ -86,6 +86,8 @@ typedef struct cmark_node cmark_node;
 typedef struct cmark_parser cmark_parser;
 typedef struct cmark_iter cmark_iter;
 
+typedef void (*CMarkNodeUserDataFreeFunc) (void *user_data);
```

Comment:
Can you please stick to snake case? A short name like `cmark_free_func` would be OK with me.